test/jdk/java/nio/channels/DatagramChannel/Loopback.java

```diff
@@ -22,7 +22,7 @@
  */
 
 /* @test
- * @bug 8236184
+ * @bug 8236184 8332535
```

Comment:
You can remove 8332535 here. We usually only put bug ids here if there are source code changes that the test is testing. If the bug being fixed is in the test itself, we don't list the bug id here.
```suggestion
 * @bug 8236184
```

My tests were positive. I will approve after you make this change.

Reply:
Okey.